metadb/scrapers/spotgen.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import spotipy
+
+
+sp = spotipy.Spotify()
+sp.trace = False
+
+def get_artist(name):
+	results = sp.search(q='artist:' + name, type='artist')
+	items = results['artists']['items']
+	if len(items) > 0:
+		return items[0]
+	else:
+		return None
+
+
+
+def scrape(mbid, artist, title, metadata=None):
+	data = None
+	spot_artist = get_artist(artist)
+	print('====', spot_artist['name'], '====')
```

Comment:
You don't need print statements in the final version of this file, as it is run automatically